tests/e2e/spec/adblocker.spec.js

```diff
@@ -0,0 +1,201 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+import { browser, expect } from '@wdio/globals';
+import {
+  enableExtension,
+  setPrivacyToggle,
+  setCustomFilters,
+} from '../utils.js';
+
+import { PAGE_DOMAIN, PAGE_URL as baseUrl } from '../wdio.conf.js';
+
+const PAGE_URL = baseUrl + 'adblocker/index.html';
+const BROWSER_CONSTRAINT_FIREFOX = 1;
+const BROWSER_CONSTRAINT_CHROMIUM = 2;
+
+async function collectTestResults() {
+  let result;
+  for (let i = 0; i < 4; i++) {
+    result = await browser.execute(() => {
+      if (
+        typeof suite === 'undefined' ||
+        suite.collection.expected !== suite.collection.reports.length
+      ) {
+        return false;
+      }
+      return suite.collection.reports;
+    });
+    if (result === false) {
+      await new Promise(function (resolve) {
+        setTimeout(function () {
+          resolve(null);
+        }, 1000);
+      });
+    } else {
+      return result;
+    }
+  }
+
+  expect(result).not.toBe(false);
+
+  return result;
+}
+
+describe.only('Adblocker Capabilities', function () {
```

Comment:
This `.only` flag is set to quickly assess the results but this should be removed before the pull request merges.